contract_manager/src/contracts/evm.ts

```diff
@@ -163,9 +268,32 @@ export class EVMContract extends Contract {
     };
   }
 
-  async executeGovernanceInstruction(privateKey: string, vaa: Buffer) {
-    const web3 = new Web3(this.chain.rpcUrl);
-    const { address } = web3.eth.accounts.wallet.add(privateKey);
+  async executeUpdatePriceFeed(senderPrivateKey: string, vaas: Buffer[]) {
```

Comment:
fwiw i like this passing in the price updates version better than the one above that queries the price service internally